src/constants/wallet.ts

```diff
@@ -32,27 +31,32 @@ export const SUPPORTED_WALLETS: { [key: string]: WalletInfo } = {
     primary: true,
   },
   METAMASK: {
-    connector: injected,
-    wallet: Wallet.INJECTED,
+    connector: metaMask,
     name: 'MetaMask',
     iconURL: METAMASK_ICON_URL,
     description: 'Easy-to-use browser extension.',
     href: null,
     color: '#E8831D',
   },
+  TALLY: {
+    connector: tally,
+    name: 'TallyHo',
+    iconURL: TALLY_ICON_URL,
+    description: 'Decentralized, community owned web3 wallet.',
+    href: null,
+    color: '#E8831D',
+  },
   WALLET_CONNECT: {
     connector: walletConnect,
-    wallet: Wallet.WALLET_CONNECT,
     name: 'WalletConnect',
     iconURL: WALLETCONNECT_ICON_URL,
     description: 'Connect to Trust Wallet, Rainbow Wallet and more...',
     href: null,
     color: '#4196FC',
     mobile: true,
   },
-  COINBASE_WALLET: {
+  WALLET_LINK: {
```

Comment:
@greg-nagy I've just noticed this and I think it should be still Coinbase, right?

Reply:
Can't remember, but probably.